src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-form/service-form.component.html

```diff
@@ -1203,6 +1203,24 @@
             </div>
           </div>
         </ng-container>
+
+      <!-- RGW QAT Compression -->
+        @if(serviceForm.controls.service_type.value === 'rgw') {
+        <div class="form-item"
+             formGroupName="qat">
+          <cds-select formControlName="compression"
+                      name="compression"
+                      label="QAT compression mode"
+                      id="compression"
+                      helperText="QAT compression is optional. Choose Hardware or Software to enable compression, or select None to disable it."
```

Comment:
NIT: I know this was provided by doc, but personally I don't think there is any point on mentioning that the field is optional on the helper text since we add required/optional to label fields already (whenever necessary).